docker/Dockerfile.xenial_debug

```diff
@@ -19,6 +22,6 @@ RUN apt-get update && \
       elvis-tiny \
       lsof \
       linux-tools-generic \
-      sudo &&  apt-get upgrade -y && \
-    rm -rf /var/lib/apt/lists/*
-
+      sudo && apt-get upgrade -y && \
+      apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /var/log/*log /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old
```

Comment:
I built the master container.  The following directories are empty and hence no rm -rf is needed.  IOW they can be removed from the PR:

```
/tmp/
/var/lib/apt/lists/
/var/tmp/
```

The rest seem valid and produce a 1MB reduction when removed.

If you want a larger reduction (~5MB), you might take a look at this pattern we found in the Kolla community several years ago:
https://github.com/openstack/kolla/blob/master/docker/base/dpkg_reducing_disk_footprint

https://github.com/openstack/kolla/blob/master/docker/base/Dockerfile.j2#L256